packages/authorization/src/authorize-interceptor.ts

```diff
@@ -42,6 +42,7 @@ export class AuthorizationInterceptor implements Provider<Interceptor> {
     this.options = {
       defaultDecision: AuthorizationDecision.DENY,
       precedence: AuthorizationDecision.DENY,
+      defaultStatusCodeForDeny: 403,
```

Comment:
This is a breaking change. Please add the following to the commit log:

```
feat(authorization): set default HTTP status code for authorization error to 403

BREAKING CHANGE: We now use http status code `403` instead of `401` to report authorization denied. The default status code can be set via `defaultStatusCodeForDeny` for the authorization options.
```